dataeng/jobs/analytics/SnowflakeDemographicsCleanup.groovy

```diff
@@ -0,0 +1,58 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+
+
+class SnowflakeDemographicsCleanup {
+    public static def job = { dslFactory, allVars -> 
+        dslFactory.job("snowflake-demographics-cleanup") {
+            logRotator common_log_rotator(allVars)
+            parameters secure_scm_parameters(allVars)
+            parameters {
+                stringParam('ANALYTICS_TOOLS_URL', allVars.get('ANALYTICS_TOOLS_URL'), 'URL for the analytics tools repo.')
+                stringParam('ANALYTICS_TOOLS_BRANCH', allVars.get('ANALYTICS_TOOLS_BRANCH'), 'Branch of analtyics tools repo to use.')
+            }
+            environmentVariables {
+                env('KEY_PATH', allVars.get('KEY_PATH'))
+                env('PASSPHRASE_PATH', allVars.get('PASSPHRASE_PATH'))
+                env('USER', allVars.get('USER'))
+                env('ACCOUNT', allVars.get('ACCOUNT'))
+            }
```

Comment:
Same thing here -- these are all NULL.  Curious whether it was a conscious decision to make these environment variables instead of stringParam values.

Reply:
I was basing this off of prior art (`SnowflakePublicGrantsCleaner.groovy`). This was also a similar pattern that I used when I was writing the groovy for the other job (`SnowflakeMicrobachelorsITK.groovy`) I previously  worked on.

Would it be better to ditch the environmentVariables section all together and add them all as stringParams?